src/GraphQL.Analyzers.CodeFixes/FieldBuilderCodeFixProvider.cs

```diff
@@ -77,6 +77,17 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
             nameArg!.Expression,
             typeArg?.Expression);
 
+        const int oneLevelIndentation = 4;
+        int fieldInvocationIndentation = newFieldInvocationExpression
+            .GetLeadingTrivia()
+            .Where(trivia => trivia.IsKind(SyntaxKind.WhitespaceTrivia))
+            .Sum(trivia => trivia.FullSpan.Length);
+
+        var whitespaceTrivia = Whitespace(new string(' ', fieldInvocationIndentation + oneLevelIndentation));
+        var newLineTrivia = RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+            ? CarriageReturnLineFeed
+            : LineFeed;
```

Comment:
Is this not equivalent to `Environment.NewLine` ?

https://learn.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-7.0

> `\r\n` for non-Unix platforms, or `\n` for Unix platforms.

Reply:
Will try this
```c#
EndOfLine(Environment.NewLine)
```